src/lndk_offers.rs

```diff
@@ -123,6 +132,50 @@ impl OfferHandler {
         .await
         .unwrap()
     }
+
+    // create_reply_path creates a blinded path to provide to the offer maker when requesting an
+    // invoice so they know where to send the invoice back to.
```

Comment:
nit(s): use `///` for documentation (I know we're all over the place in this repo with this convention, but worth trying to do). 

+ Can we add a comment on the strategy for this? That we try to get a peer and fall back to ourselves?